packages/meteor-developer-config-ui/package.js

```diff
@@ -0,0 +1,13 @@
+Package.describe({
+  summary: 'Blaze configuration templates for the Meteor developer accounts OAuth.',
+  version: '1.0.0-beta.1'
```

Comment:
Yes, I kept it that way since it was in the approved [`twitter-config-ui` package as well](https://github.com/meteor/meteor/pull/8283/files#diff-01a082e96697848e609f811415664e1e). I agree though, it doesn't seem to be needed. I'll remove it from here and the `github-config-ui` package. Thanks @laosb!